### PR TITLE
Feature/meeting page

### PR DESCRIPTION
**Pull request by krausweb**
- added the main layout and styles
- added the meeting route for the meeting page
- fixed link from calendar list to the meeting page
- added real meeting date
- added new icons
- fixed icon styles
- added layout and styles on the mobile version

### CODE REVIEW

**Comment by krausweb** [comment]
@MBeckers please,
I need real data for
**Sorting and Search functionality**
I don't have "funktion" and "party" fields.

See temporary method getMeeting
/app/Http/Controllers/MeetingController.php@getMeeting
/resources/views/meeting.blade.php

File: resources/js/components/Meeting.vue [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+            }
+        },
+        collapseAgendaChild(agendaChildNumber, agendaType, agendaParentRef) {
+            this.agendaChildNumber = agendaChildNumber;
+            this.agendaType = agendaType;
+            this.agendaParentRef = [];
+            this.backButtonPublic = false;
+            this.backButtonPrivate = false;
+            this.isActiveParent = this.isActiveParent === false;
+            //this.isActiveChild = this.isActiveChild === false;
```
Comment (MBeckers) [inline resources/js/components/Meeting.vue]: remove please

File: resources/js/components/Meeting.vue [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+                // hide all agenda list(if 2 or more)
+                /*agendaList.forEach(agenda => {
+                    agenda.classList.toggle('ris-agenda-list_hidden');
+                });*/
+
+                //const agendaHeadParent = agendaHead.parentElement;
+                // show only list block where chosen items
+                // agendaHeadParent.classList.toggle('ris-agenda-list_hidden');
+                // agendaHeadParent.classList.toggle('ris-agenda-list_active');
+
```
Comment (MBeckers) [inline resources/js/components/Meeting.vue]: remove please if not needed

File: resources/js/components/Meeting.vue [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+                    this.agendaList.forEach(agenda => {
+
+                        let agendaMainNumber = agenda.number.split('.', 1)[0];
+                        console.log(agendaChildNumber, +agendaMainNumber);
+
+                        if (agenda.number && agenda.number.includes('.')
+                            && agendaChildNumber === +agendaMainNumber) {
+
+                            console.log('Add child: ', agenda.id, agendaChildNumber, agendaMainNumber);
+                            this.agendaChildRef[agenda.id] = true;
```
Comment (MBeckers) [inline resources/js/components/Meeting.vue]: do we need the console.log?

**Comment by krausweb** [comment]
All tabs compleated and fixed. 
Please, need to approve.

**Comment by MBeckers** [comment]
we still have console logs, when opening tabs

image

we should get rid of this

**Comment by krausweb** [comment]
Hm, strange, please generate a new app.js (yarn dev)

**Pull request merged**